tasks/main.yml

```diff
@@ -1,5 +1,20 @@
 ---
 
+- name: check for Debian family system
+  stat:
+    path: '/etc/debian_version'
+  register: reg_puppetize_os_debian
+
+- name: set os family fact
+  set_fact:
+    os_family: 'Debian'
+  when: reg_puppetize_os_debian.stat.exists == True
+
+- name: gather os specific variables
+  include_vars: "{{ item }}"
+  with_first_found:
```

Comment:
Do we need the `stat` and `set_fact` tasks? I think it should be possible to get the same result by only using `include_vars` https://docs.ansible.com/ansible/latest/modules/include_vars_module.html (the third example)` , the end result would IMO be more elegant.

Reply:
hi @oscarkraemer ,
How does that example differ from the reverted changes proposed in #17? This Idiom is in use due to the conversation in #19 where it was decided there shouldn't be fact gathering.

thanks,
Karl.

Reply:
I can see how I put you between a rock and a hard place. In that case I would like to ask you to add the `tags: ['install_puppet_client']` to the new tasks, and then I can do a squash-merge.

Long-term, I'm thinking that we would benefit of allowing gather_facts when we want to install puppet. Since I think the main reason why we want to be able to run the playbook without gathering facts is to make the playbook run as fast as possible with the `puppet_run_only: true`.

Reply:
I managed to add the tag now to goetzk's role/branch, let's see if travis passes :)

Reply:
Hi @oscarkraemer,
I just got back to your request but I see @martbhell has already done it. 

thanks
karl.